packages/realm-server/scripts/start-drafts-root.sh

```diff
@@ -1,6 +1,20 @@
 #! /bin/sh
+check_postgres_ready() {
+  docker exec boxel-pg pg_isready -U postgres >/dev/null 2>&1
+}
+# remove this check after the feature flag is removed
+if [ -n "$PG_INDEXER" ]; then
+  while ! check_postgres_ready; do
+    printf '.'
+    sleep 1
+  done
+fi
```

Comment:
Since this is repeated across a few scripts, maybe it should be extracted?

Reply:
Sounds good